src/V1/Actor/Template/Builder.php

```diff
@@ -32,20 +32,38 @@ public function build(): TemplateInterface
     protected function getFilePath(): string
```

Comment:
method: "I ask for a possibly empty array of Template file path candidates. I find the first Template in the stack that I can validate exists on the FS and store it as a property on my associated object. If I cannot validate that any in the array exist on the FS I throw an exception. If I stored one on my associated object, I return it to the caller."